lib/performance_platform/gateway/performance_report.rb

```diff
@@ -24,4 +24,14 @@ def post(uri, data)
       JSON.parse(response.body)
     end
   end
+
+  def build_url(data)
+    URI("#{ENV['PERFORMANCE_URL']}data/gov-wifi/#{data[:metric_name]}")
```

Comment:
I always opt for .fetch over [
Just to avoid a nil getting in there if the ENV isn't set correctly

Reply:
Do we want this *not* to blow up when ENV variables are wrong?

Reply:
(I mean, I can see both approaches as legitimate strategies.)